src/vs/editor/contrib/find/browser/simpleFindWidget.ts

```diff
@@ -21,13 +29,20 @@ const NLS_NEXT_MATCH_BTN_LABEL = nls.localize('label.nextMatchButton', "Next mat
 const NLS_CLOSE_BTN_LABEL = nls.localize('label.closeButton', "Close");
 
 export abstract class SimpleFindWidget extends Widget {
+
 	protected _findInput: FindInput;
 	protected _domNode: HTMLElement;
 	protected _isVisible: boolean;
 	protected _focusTracker: dom.IFocusTracker;
+	protected _findInputFocusTracker: dom.IFocusTracker;
+	protected _findInputFocused: IContextKey<boolean>;
+	protected _findHistory: HistoryNavigator<string>;
+	protected _updateHistoryDelayer: Delayer<void>;
 
 	constructor(
 		@IContextViewService private _contextViewService: IContextViewService,
+		@IContextKeyService private _contextKeyService: IContextKeyService,
+		@ISimpleFindWidgetService private _simpleFindWidgetService: ISimpleFindWidgetService,
 		private animate: boolean = true
```

Comment:
I know there are inconsistencies, try to stick to the style of the file, all the properties and constructor args above use the _ prefix